### DISCUSSION TITLE
loadfile with start not working

### DISCUSSION THREAD

**Discussion by emveepee** [mpv-player/mpv#14986] [Q&A]
I have a libmpv app and using mpv-dev-x86_64-v3-20240929-git-c3d9243.7z when I try to use loadfile with start=## option it doesn't work and I get a -MPV_ERROR_INVALID_PARAMETER  -4 Has the ability to resume when starting a video from the library been removed?

**Comment by kasper93** [comment]
See https://github.com/mpv-player/mpv/commit/c678033c1d60b48ae02fbbe4815869b9504a17f6 the `loadfile` has been changed. There was discussion about it here https://github.com/mpv-player/mpv/discussions/13974

**Reply by emveepee** [reply]
OK thanks I did try and read that but it is really difficult to understand and I didn't think replace changed 

```
		std::string option = fmt::format("start={}",position);
		const char* cmd[] = { "loadfile", filename, "replace", "0", option.c_str(), NULL};
		int ret = mpv_command(mpv, cmd);
		if (ret == MPV_ERROR_INVALID_PARAMETER)
		{
			const char* cmd[] = { "loadfile", filename, "replace", option.c_str(), NULL };
			ret = mpv_command(mpv, cmd);
		}
```

I guess this works to support user's dll's but it really odd to add a bogus variable.
